### DISCUSSION TITLE
Broken defsec-links in docs

### DISCUSSION THREAD

**Discussion by thomasroot** [aquasecurity/trivy-operator#1778] [Help & Support] [answered]
Good morning,

I just noticed that the "defsec"-link in https://aquasecurity.github.io/trivy-operator/v0.18.1/docs/configuration-auditing/built-in-policies/ are broken. I would have tried to create a PR myself but I couldn't find the cisbenchmark-rules in the github repo. 

Best regards,
Thomas

**Comment by AnaisUrlichs** [comment]
This is for @simar7 -- do you know where the information is? 😄

**Reply by simar7** [reply, marked as answer]
That should be here https://github.com/aquasecurity/trivy-policies/tree/main/checks/kubernetes/cisbenchmarks
